pro/src/new_components/BoxFormLayout/BoxFormLayoutFormHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import cn from 'classnames'
+import React from 'react'
+
+import style from './BoxFormLayout.module.scss'
+
+interface BoxFormLayoutFormHeader {
```

Comment:
```suggestion
interface IBoxFormLayoutFormHeaderProps {
```
pour être raccord avec ailleurs dans le code